test/email_reply_parser_test.rb

```diff
@@ -7,6 +7,7 @@
 require dir + '..' + 'lib' + 'email_reply_parser'
 
 EMAIL_FIXTURE_PATH = dir + 'emails'
+SF_EMAIL_FIXTURE_PATH = dir + 'sf_emails/fr'
```

Comment:
Looks like this is unused

Reply:
Oops; not sure why my editor didn't highlight that this time.  Good eye!  Done!